starter-code/javascript/main.js

```diff
@@ -1,20 +1,37 @@
 var chronometer = new Chronometer();
-var btnLeft     = document.getElementById('btnLeft');
-var btnRight    = document.getElementById('btnRight');
-var minDec      = document.getElementById('minDec');
-var minUni      = document.getElementById('minUni');
-var secDec      = document.getElementById('secDec');
-var secUni      = document.getElementById('secUni');
-var milDec      = document.getElementById('milDec');
-var milUni      = document.getElementById('milUni');
+var btnLeft = document.getElementById('btnLeft');
+var btnRight = document.getElementById('btnRight');
+var minDec = document.getElementById('minDec');
+var minUni = document.getElementById('minUni');
+var secDec = document.getElementById('secDec');
+var secUni = document.getElementById('secUni');
+var milDec = document.getElementById('milDec');
+var milUni = document.getElementById('milUni');
 
 
+var crono = new Chronometer();
+
 function printTime() {
 
+
+
+    setInterval(function () {
+
+        crono.setMinutes();
+        crono.setSeconds();
+
+        minUni.innerHTML = crono.currentMinutes;
+        secUni.innerHTML = crono.currentSeconds;
+
+        // Arrived until here, couldn't get the seconds correctly.
```

Comment:
You have 
```
+var minDec = document.getElementById('minDec');
+var minUni = document.getElementById('minUni');
+var secDec = document.getElementById('secDec');
+var secUni = document.getElementById('secUni');
```
So you have to put for the minutes and the seconds each character on each element